librosa/feature/spectral.py

```diff
@@ -1163,6 +1164,104 @@ def logfsgram(y=None, sr=22050, S=None, n_fft=4096,
     return cq_basis.dot(S)
 
 
+@cache
+def tonnetz(y=None, sr=22050, chromagram=None, norm=np.inf):
+    '''Computes the tonal centroid features (tonnetz).
+
+    Parameters
+    ----------
+    y : np.ndarray [shape=(n,)] or None
+        Audio time series.
+
+    sr : int > 0 [scalar]
+        sampling rate of `y`
+
+    chromagram : np.ndarray [shape=(n_chroma, t)] or None
+        Normalized energy for each chroma bin at each frame.
+
+        If `None`, a cqt chromagram is performed.
+
+    norm : float or None
+        Column-wise normalization.
+        See `librosa.util.normalize` for details.
+
+        If `None`, no normalization is performed.
+
+    Returns
+    -------
+    ton : np.ndarray [shape(6, t)]
+        Tonal centroid features for each frame.
+        Tonnetz dimensions:
```

Comment:
What are 'x' and 'y' here?

I ask because I'm thinking it would be nice to add a tonnetz axis labeler to `display.specshow`, so you can do something like

```python
>>> ton = librosa.feature.tonnetz(y=y, sr=sr)
>>> librosa.display.specshow(ton, y_axis='tonnetz')
```
and get the right thing.